src/TinyUPnP.cpp

```diff
@@ -960,45 +1078,33 @@ String TinyUPnP::getPath(String url) {
   }
   int firstSlashIndex = url.indexOf("/");
   if (firstSlashIndex == -1) {
-	debugPrintln("ERROR: Cannot find path in url [" + url + "]");
+	UPNP_DEBUGln("ERROR: Cannot find path in url [" + url + "]");
 	return "";
   }
   return url.substring(firstSlashIndex, url.length());
 }
 
-String TinyUPnP::getTagContent(String line, String tagName) {
-  int startIndex = line.indexOf("<" + tagName + ">");
+String TinyUPnP::getTagContent(String &line, String tagName, int startIndex) {
   if (startIndex == -1) {
-	debugPrint(F("ERROR: Cannot find tag content in line ["));
-	debugPrint(line);
-	debugPrint(F("] for start tag [<"));
-	debugPrint(tagName);
-	debugPrintln(F(">]"));
+		startIndex = line.indexOf("<" + tagName + ">");
+	}
+  if (startIndex == -1) {
```

Comment:
not sure I get what's going on here.

Reply:
I didn't change this bit of code .. it's the same as the old one ..
the lines numbers changed ..